examples/compat/ggplot/density.py

```diff
@@ -9,4 +9,4 @@
 
 plt.title("Density ggplot-based plot in Bokeh.")
 
-mpl.to_bokeh(name="density")
+mpl.show(mpl.to_bokeh(name="density"))
```

Comment:
Not strong preferences... if you want to change it @mattpap, let me know... if not, I will merge this one in a couple of hours.